src/eth/rpc.ts

```diff
@@ -53,22 +53,22 @@ export async function getSnapshotRound ():Promise<BigNumber> {
         } else {
             return await roundsManager.lipUpgradeRound(BigNumber.from(52))
         }
-    } catch (err) {
+    } catch (err: any) {
         return err
     }
 }
 
-export async function getEarningsRoot() {
+export async function getEarningsRoot(LIPRound: string) {
```

Comment:
```suggestion
export async function getEarningsRoot(LIP: string) {
```

since the value is string of the format LIP-X and not a round number.